cfme/tests/webui/test_general_ui.py

```diff
@@ -509,6 +512,44 @@ def test_provider_details_page_refresh_after_clear_cookies(
     assert login_view.is_displayed
 
 
+@pytest.mark.tier(1)
+@pytest.mark.provider([InfraProvider], selector=ONE_PER_CATEGORY)
+@pytest.mark.meta(automates=[1733120], blockers=[BZ(1733120), BZ(1784179)])
+def test_compare_vm_from_datastore_relationships(appliance, setup_provider, provider):
+    """
+    Bugzilla:
+        1733120
+        1784179
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Infra
+        caseimportance: medium
+        initialEstimate: 1/18h
+        setup:
+            1. Add an infra provider.
+        testSteps:
+            1. Select a datastore with at least 2 VMS, and navigate to a it's Details page.
+            2. Click on Managed VMs from the relationships table.
+            3. Select at least 2 VMs and click on `Configuration > Compare the selected items`
+        expectedResults:
+            1.
+            2.
+            3. Comparison page should be displayed, there should be no exception on the page.
+    """
+    datastore = appliance.collections.datastores.instantiate(
+        name=provider.data["provisioning"]["datastore"], provider=provider
+    )
+    view = navigate_to(datastore, "ManagedVMs")
+
+    # Check any 3 entities for comparison
+    [vm.check() for vm in sample(view.entities.get_all(), 3)]
+
+    view.toolbar.configuration.item_select("Compare Selected items")
+    compare_view = datastore.create_view(DatastoresCompareView)
+    assert compare_view.is_displayed
```

Comment:
Note: Currently the page lands in an exception.

Reply:
@valaparthvi Is it a bug or automation issue?

Reply:
It's a bug.

Reply:
Then, I would  suggest to report a BZ and block this test case?

Reply:
It is already reported, this test case covers that BZ.

Reply:
Still failing on PRT. Please check or add **forced_streams**.

Reply:
511 failure is unrelated to these changes. Moving it to RFR.